src/main/java/net/sf/javaanpr/imageanalysis/HoughTransformation.java

```diff
@@ -33,40 +33,40 @@
     private int height;
 
     public HoughTransformation(int width, int height) {
-        this.maxPoint = null;
-        this.bitmap = new float[width][height];
+        maxPoint = null;
+        bitmap = new float[width][height];
         this.width = width;
```

Comment:
Lets make `width` and `height` final. I know this wasn't the aim of your PR, but it will be way easier to reason about all future changes this way.